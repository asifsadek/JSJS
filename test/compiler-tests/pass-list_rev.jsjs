val a = [1, 2, 3, 4, 5];

List.print_list(List.rev(a));

val b = ["one", "two", "three", "four", "five"];

List.print_list(List.rev(b));

val c = [true, false, false, true, false];

List.print_list(List.rev(c));
