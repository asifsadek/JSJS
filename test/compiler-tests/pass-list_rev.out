5
4
3
2
1
five
four
three
two
one
false
true
false
false
true
